src/connectors/autocomplete/connectAutocomplete.ts

```diff
@@ -194,6 +188,4 @@ search.addWidgets([
       },
     };
   };
-};
-
-export default connectAutocomplete;
+} as Connector<AutocompleteRendererOptions, AutocompleteConnectorParams>);
```

Comment:
could we make this instead:

```
const connectAutocomplete: Connector<AutocompleteRendererOptions, AutocompleteConnectorParams> = function connectAutocomplete() {};
export default connectAutocomplete;
```

(or even `export const` so we don't need to rename default back to the variable later)

thanks for suggesting this @yannickcr

Reply:
I also find the variable syntax less awkward than the function "cast".

In favor of `export const` as well, if you believe this is the right PR to do so. More generally we should aim at reducing the amount of default exports in the codebase.

Reply:
Personally, I found the way I did more clean ( less typing too ).

Reply:
I always thought that `as` was a bad practice in typescript, but I might have made it up, since I don't see any reason why it's different